server_development/topics/auth-spi.adoc

```diff
@@ -798,7 +798,8 @@ a different credential or execution.
 Adding an Authenticator to a flow must be done in the admin console.
 If you go to the Authentication menu item and go to the Flow tab, you will be able to view the currently defined flows.
 You cannot modify built in flows, so, to add the Authenticator we've created you have to copy an existing flow or create your own.
-I'm hoping the UI is intuitive enough so that you can figure out for yourself how to create a flow and add the Authenticator.
+I'm hoping the UI is intuitive enough so that you can figure out for yourself how to create a flow and add the Authenticator. For
```

Comment:
Change "I'm hoping the UI is intuitive enough so that you can figure out for yourself how to" 
to
"Our hope is that the user interface is sufficiently clear so that you can determine how to"

Reply:
Fixed